src/BlockListener.php

```diff
@@ -38,7 +38,7 @@ function (&$index, Line $line) use ($pick, &$first) {
                 // its the same line as the start.. skip this one as its by default included in while operations
                 if ($line === $pick->line) {
                     return true;
-                } elseif (($line->hasEndNewline() || $line->hasNewline())) {
+                } elseif (($line->hasEndNewline() || $line->hasNewline() || ($line->isJsonInsert() && !$line->isInline()))) {
```

Comment:
@jwissels as you've dived into this area recently, do you see any issues adding this?

Reply:
I have not found problems with this yet